test/utils/logger.test.js

```diff
@@ -231,13 +231,22 @@ describe('#processLogData', () => {
 describe('#logFormat', () => {
   const mockTimestamp = new Date().toISOString();
   const mockLogLevel = faker.random.arrayElement(['error', 'warn', 'info', 'debug', 'verbose']);
+  const mockMessage = faker.lorem.sentence();
+
+  it('logs with correct format', () => {
+    const formattedLog = logFormat({ level: mockLogLevel, message: mockMessage, timestamp: mockTimestamp });
+
+    const expectedLog = `[${mockTimestamp}] [${LOG_PREFIX} ${mockLogLevel.toUpperCase()}] [${mockMessage}]`;
+
+    expect(formattedLog).toEqual(expectedLog);
+  });
```

Comment:
Optional: Can we drop this below the following test? I suppose they both have utility, but having this one above the subsequent test makes the latter seem slightly silly. I'm somewhat inclined to merge them, but perhaps not.

Reply:
Sure. I can do that